app/assets/javascripts/shared/routes.js

```diff
@@ -110,6 +110,8 @@ class APIRoutes {
   static getSubsectionPath(id)            { return APIRoutes.createRoute(`students/subsections/${id}`) }
   static getComponentProgressPath(id)     { return APIRoutes.createRoute(`students/components/${id}/component_progresses`) }
   static createComponentProgressPath(id)  { return APIRoutes.createRoute(`students/components/${id}/component_progresses`)}
+  static finishedCoursePath(id)           { return APIRoutes.createRoute(`students/student_courses/${id}`)}
+  static getStudentCourseIdPath(id)       { return APIRoutes.createRoute(`students/student_courses/${id}`)}
 
```

Comment:
Nit: pls add spaces between `)` and `}` for lines 112-114 and wherever else applicable (you can get more lines now!!)